### ISSUE TITLE
github output for Exceptions?

### ISSUE THREAD

**Issue by muncus** [open-policy-agent/conftest#782]
When using `--output github` (specifically in the context of Github Actions), exceptions are only shown in the detailed output:

```
Testing 'repository.json' against 7 policies in namespace 'github.repo'
  exception file=repository.json data.github.repo.exception[_][_] == "merge_commits"
  success file=repository.json 6
7 tests, 6 passed, 0 warnings, 0 failures, 1 exception
```
(taken from this [example run](https://github.com/muncus/conftest-demo/actions/runs/4238809229))

Warnings and Errors are surfaced as Github Annotations, which are much more prominent in the UI.

Can we surface Exceptions as a [Notice](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-notice-message) to make them more visible? We may want put this behind an opt-in flag, so the default behavior does not change.

I'm willing to create the PR if there's agreement on the feature.

**Comment by jalseth** [comment]
This makes sense to me, and it should be pretty straightforward to add. I'd welcome a PR!

Relevant code: https://github.com/open-policy-agent/conftest/blob/c9fbd26ce6880ec413cabfdfdd30af634077b134/output/github.go#L45

**Comment by shayyxi** [comment]
I can work on this.

**Comment by boranx** [comment]
Seems to be fixed in #818 , feel free to re-open if you think it's not
